types/hapi/test/route/additional-options.ts

```diff
@@ -6,6 +6,29 @@ var authConfig: Hapi.RouteAdditionalConfigurationOptions = {
   app: {}
 };
 
+var extConfigSingle: Hapi.RouteAdditionalConfigurationOptions = {
+  ext: {
+    type: 'onPreAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }
+}
+
+var extConfigMulti: Hapi.RouteAdditionalConfigurationOptions = {
+  ext: [{
+    type: 'onPreAuth',
+    method: <Hapi.ServerExtRequestHandler> function (request, reply) {
+      reply('ok');
+    }
+  }, {
```

Comment:
Would you mind also adding a test for a server ext point too?

    {
        type: 'onPostStart',
        method: <Hapi.ServerExtFunction> function (server, next) {
            next();
        }
    }, {

We'd need to also change this:

```ts
/**
 * Used by server extension points
 * err can be BoomError or Error that will be wrapped as a BoomError
 * For source [See code](https://github.com/hapijs/hapi/blob/v16.1.1/lib/reply.js#L109-L118)
 * For source [See code](https://github.com/hapijs/hapi/blob/v16.1.1/lib/response.js#L60-L65)
 */
export interface ContinuationFunction {
    (err?: Boom.BoomError): void;
}
```

(the castings annoying isn't it but I can't figure a way around it yet, I had hoped the string literals on the type would force the inference of the method signature.

(Can't seem to add it as a normal comment, only lets me add a review comment, we don't have to add this).